artiq/coredevice/comm_moninj.py

```diff
@@ -21,6 +17,16 @@ class TTLOverride(Enum):
     oe = 2
 
 
+class ICommMonInj:
+    def monitor_probe(self, enable: int, channel: int, probe: int): raise NotImplementedError
+
+    def monitor_injection(self, enable: int, channel: int, overrd: int): raise NotImplementedError
+
+    def inject(self, channel: int, override: int, value: int): raise NotImplementedError
+
+    def get_injection_status(self, channel: int, override: int): raise NotImplementedError
+
+
```

Comment:
See comments on previous PR, you most likely don't need to change this file at all and again, type annotations if you want them should be in a separate PR, applied to a whole module at a time, and checked for correctness.